native/src/main/java/io/ballerina/stdlib/graphql/runtime/engine/ArgumentHandler.java

```diff
@@ -107,6 +124,12 @@ private Object getArgumentValue(BObject argumentNode, Type parameterType) {
         }
     }
 
+    private BMap<BString, Object> getRepresentationArgument(Object jsonRepresentation, Type parameterType) {
+        BMap<BString, ?> map = JsonUtils.convertJSONToMap(jsonRepresentation, PredefinedTypes.TYPE_MAP);
```

Comment:
Can't we use `Object` here?

```suggestion
        BMap<BString, Object> map = JsonUtils.convertJSONToMap(jsonRepresentation, PredefinedTypes.TYPE_MAP);
```